### ISSUE TITLE
Create org chart in HTML Block in Wagtail

### ISSUE THREAD

**Issue by johnnyporkchops** [fecgov/fec-cms#3185]
### Summary

**What we're after:**
Until there is a Wagtail template to create/edit the org chart in Wagtail, we can use the html in this [issue 3130](https://github.com/fecgov/fec-cms/pull/3130) to publish this in the mean time.
### Related issues
https://github.com/fecgov/fec-cms/pull/3130
https://github.com/fecgov/fec-cms/issues/3072

### Completion criteria

_List action items that need to be done to complete this task. List in checklist formatting._
- [x] Merge https://github.com/fecgov/fec-cms/pull/3130
- [x] Once full-width template is merged (https://github.com/fecgov/fec-cms/pull/3188) Build org chart in html block in full-width page (Pair with content team on this so that they understand how to do this)
- [x] Second pair of eyes
- [x] Publish page
- [x] Update https://www.fec.gov/about/leadership-and-structure/ to point to new page
- [x] Put in redirect for old organization chart

### Future work
#3072

**Comment by dorothyyeager** [comment]
Questions for @johnnyporkchops @rfultz @patphongs - 

1) I found the template but it's not available to us in the "About the FEC section" (which is where we'd want it. Do we need a ticket for that work?

2) The template doesn't seem to be available at all in prod, just feature. Do we need to have that happen first before I can work on this ticket?

3) The footnotes in the org chart aren't in the HTML. When I tested out coding them in feature, I cannot get a space to appear between the chart and the footnotes. I tried putting them into separate HTML block and the same block. No luck.

4) The link for the IG office in the HTML code provided was out of date. I updated it in my text file; should I do the same in the hard-coded file Pat linked me to?

**Comment by dorothyyeager** [comment]
I've published the new chart at https://www.fec.gov/about/leadership-and-structure/fec-offices/ and updated the link on the "Leadership and structure" page. I've also added the new page to our redirect spreadsheet and we'll tackle it in the redirects at #3209 . We're ready to close this ticket.
